src/pages/pr/collage_template.tsx

```diff
@@ -0,0 +1,110 @@
+import { Box, Divider, HStack, Image, Text, VStack } from "@chakra-ui/react";
+
+interface CollageProps {
+    title: string;
+    locations: string[];
+    durations: number[];
+    imageUrls: string[];
+    introduction: string;
+}
+
+export function CollageTemplate({
+    title,
+    locations,
+    durations,
+    imageUrls,
+    introduction,
+}: CollageProps) {
+    return (
+        <Box padding={16}>
+            <Box textAlign="center" mb={4}>
+                <Text
+                    color="rgba(130, 141, 205, 1)"
+                    fontFamily="Inter"
+                    fontWeight="bold"
+                    fontSize="80px"
+                >
+                    {title}
+                </Text>
+            </Box>
+            <Divider borderColor="#A4ABD4" my={4} />
+            {locations.map((location, index) => (
+                <Box key={index}>
+                    {index % 2 === 0 ? (
+                        <HStack
+                            justifyContent="space-between"
+                            alignItems="flex-start"
+                        >
```

Comment:
@wdtkm0823 こうすると一つにまとめられそうです
```suggestion
                        <HStack
                            justifyContent="space-between"
                            alignItems="flex-start"
                            flexDirection={index % 2 === 0 ? "row" : "row-reverse"} 
                        >
```
参考：https://developer.mozilla.org/ja/docs/Web/CSS/flex-direction